onlineapi/src/main/kotlin/com/codingblocks/onlineapi/api/OnlineJsonApi.kt

```diff
@@ -352,4 +353,24 @@ interface OnlineJsonApi {
         @Query("exclude") exclude: String = "spin_prize.*",
         @Query("include") include: String = "user,spin_prize"
     ): Response<JSONAPIDocument<List<Spins>>>
+
+    @GET("user_course_wishlists")
+    suspend fun getWishlist(
```

Comment:
<img width="323" alt="Screenshot 2020-06-14 at 7 32 13 PM" src="https://user-images.githubusercontent.com/29139786/84595484-c3f15080-ae75-11ea-9cf8-cbd9b9ee7cb1.png">
Also, the design would look like this

Reply:
Also makes this a PagedList

Reply:
Sir on website, where wishlist is shown, their is no button to remove wishlist so I removed from app too